analysis/spectra_chisq/_model_spectra.py

```diff
@@ -0,0 +1,475 @@
+"""The ``_model_spectra`` module models spectra by varying the dust extinction from the Milky Way.
+
+The model used is ``sncosmo.CCM89Dust``. The parameters A_v and amplitude are fit to the observed spectra. The best fit
+parameters are determined by ``scipy.optimize.curve_fit``
+The fit parameters are saved to a fits file.
+
+Currently, run this module at the command line using: "python _model_spectra.py"
+"""
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import sncosmo
+from astropy.table import Table
+from functools import partial
+from scipy.optimize import curve_fit
+from sndata.csp import DR1, DR3
+
+from astropy.table import join, vstack #unique
+
+#from ._synthetic_photometry import get_synth_photo_for_id # FIXME uncomment if fit performed using photometry
```

Comment:
You have a lot of commented out code. Try to avoid this. Use comments to increase clarity of what the code is doing. I find some of the code in this file hard to follow because all the commented out code makes it cluttered.